server/src/main/java/com/battleships/Player.java

```diff
@@ -0,0 +1,53 @@
+package com.battleships;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.util.Scanner;
+
+class Player {
+    private final Socket playerSocket;
+    private final PrintWriter clientWriter;
```

Comment:
You can group those together and create ClientInterface class

Reply:
Grouped into PlayerIO class